### ISSUE TITLE
The binary file created does not start.

### ISSUE THREAD

**Issue by ptfu0** [Nuitka/Nuitka#2097]
Hello. I cannot run a created file by double clicking in fluxbox, chmod +x on the file is done. It runs fine from the terminal ./my_file.bin but does not start in the window shell by double clicking, it displays a message with the choice of program to open (this was after updating nuitka but I do not remember what version I had before). This happens only on ubuntu 20.04, also it has no icon at all . On ubuntu 22.04 it works fine and the icon is there and doubleclick launches it. I did it with the command nuitka3 --standalone --plugin-enable=upx --onefile my_file.py

**Comment by kayhayen** [comment]
In the template, I have made it clear, that not filling out the issue template is not acceptable. I am closing this kind of report as invalid. The main reason is that I do not want to be left guessing. For you it's easy to erase the issue template and expect me to ask you questions only as necessary, but I cannot do that. I would much rather wait for you to create a new report, fill out the template, or wait for somebody with the necessary skill to do it.

In short: No issue template, no fun. Therefore I am likely just deleting the issue, as it serves as a bad example and detraction for searches.
